pkg/experimental/genrpm.bzl

```diff
@@ -0,0 +1,611 @@
+# Copyright 2019-2020 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Provides rules for creating RPM packages via pkgfilegroup and friends."""
+
+load("@rules_pkg//experimental:genpkg.bzl", "PackageDirInfo", "PackageFileInfo")
+
+rpm_filetype = [".rpm"]
+
+spec_filetype = [".spec", ".spec.in"]
+
+def _gen_rpm_impl(ctx):
```

Comment:
Why don't we leave it as pkg_rpm.   The genrpm was a google internal name. The other rules in the package are pkg_

Reply:
Hm.  I think I'm fine with that.  Would it also make sense to rename the file to `rpm.bzl`, or perhaps `pkg_rpm.bzl`? It's easy enough for people to do things like:

```
load("@rules_pkg//experimental:rpm.bzl", pkg_rpm_ex = "pkg_rpm")
```

Reply:
Addressed in 229a244, filename changed to `rpm.bzl`.